### ISSUE TITLE
Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?

### ISSUE THREAD

**Issue by akshat-goel** [louislam/uptime-kuma#4991]
### ⚠️ Please verify that this question has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Getting weird errors - Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?

### 🐻 Uptime-Kuma Version

1.23.13

### 💻 Operating System and Arch

Ubuntu 20.04

### 🌐 Browser

Google Chrome Version 126.0.6478.127 (Official Build) (x86_64)

### 🖥️ Deployment Environment

- Runtime: 
- Database: 
- Filesystem used to store the database on: 
- number of monitors:

**Comment by CommanderStorm** [comment]
Duplicate of [a lot of other issues](https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+Timeout+acquiring+a+connection+is%3Aclosed).

I am going to close this as resolved by `v2.0` as I don't see evidence to the contrary.

> TL;DR:
> In v1, reducing either retention/ping-rate/.. or moving to a faster storage solution is the only option to get around this problem.
> In v2, we have likely resolved this problem. See #4500 for further details of what needs to happen before we can release this version.
